faunadb-java/src/main/java/com/faunadb/client/FaunaClient.java

```diff
@@ -157,10 +169,10 @@ public FaunaClient build() {
         .withFaunaRoot(endpoint)
         .withQueryTimeout(queryTimeout)
         .withUserAgent(userAgent)
-        .withJvmDriver(JvmDriver.JAVA);
+        .withJvmDriver(JvmDriver.JAVA)
+        .withCheckNewDriverVersion(checkNewVersion);
 
       if (registry != null) builder.withMetrics(registry);
-
```

Comment:
let's return back the empty line here, we'll need to add style checker for jvm driver in future...